io.vikunja.Desktop.appdata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.vikunja.Desktop</id>
+  <name>Vikunja</name>
+  <summary>The open-source, self-hostable to-do app. Organize everything, on all platforms.</summary>
+  <project_license>AGPLv3</project_license>
```

Comment:
this is not a valid SPDX license identifier.

Reply:
Fixed